metadata-ingestion/src/datahub/ingestion/extractor/schema_util.py

```diff
@@ -317,17 +336,41 @@ def emit(self) -> Generator[SchemaField, None, None]:
                 merged_props.update(self._schema.other_props)
                 merged_props.update(schema.other_props)
 
-                tags = None
+                # Parse meta_mapping
+                meta_aspects: Dict[str, Any] = {}
+                if self._meta_mapping_processor:
+                    meta_aspects = self._meta_mapping_processor.process(merged_props)
+
+                tags = []
```

Comment:
Adding a type here would help make this more readable and also type checks.
`tags:List[?] = []`

Reply:
Done.